harness/setup.py

```diff
@@ -27,10 +27,14 @@
         "yogadl==0.1.3",
     ],
     extras_require={
-        "tf-114-cuda100": ["tensorflow-gpu==1.14.0"],
-        "tf-114-cpu": ["tensorflow==1.14.0"],
-        "pytorch-14-cuda100": ["torch==1.4.0+cu100", "torchvision==0.5.0+cu100"],
-        "pytorch-14-cpu": ["torch==1.4.0", "torchvision==0.5.0"],
+        "tf-115-cuda102": ["tensorflow-gpu==1.15.5"],
+        "tf-115-cpu": ["tensorflow==1.15.5"],
+        "tf-240-cuda102": ["tensorflow-gpu==2.4.1"],
+        "tf-240-cpu": ["tensorflow==2.4.1"],
+        "tf-241-cuda110": ["tensorflow-gpu==2.4.1"],
+        "pytorch-18-cuda102": ["torch==1.8.0", "torchvision==0.9.0"],
+        "pytorch-18-cuda110": ["torch==1.8.0", "torchvision==0.9.0"],
+        "pytorch-18-cpu": ["torch==1.7.0", "torchvision==0.8.2"],
```

Comment:
1.7 and 0.8.2?

Reply:
Oh that's wrong. I'll fix that. I might have been trying to work around a bug that I thought was due to the fact that you can't embed CUDA versions in the package name for 1.8.0 (they're going to fix that in 1.8.1). But it wasn't.